big-red-shuttle/StopsViewController.swift

```diff
@@ -101,5 +123,186 @@ class StopsViewController: UIViewController, GMSMapViewDelegate {
+            self.popUpView.frame = CGRect(x: 12, y: viewHeight - 120 , width: popUpWidth, height: 106)
+        }, completion: { _ in
+            
+            if (UserDefaults.standard.value(forKey: "nudgeCount") as! Int) < 3 && UserDefaults.standard.value(forKey: "didFireNudge") as! Bool {
+                
+                UIView.animate(withDuration: 0.2, delay: 0.3, options: .curveEaseInOut, animations: {
+                    collectionView.contentOffset.x = collectionView.contentOffset.x + 20
+                }, completion: { _ in
+                    UIView.animate(withDuration: 0.2, delay: 0.0, options: .curveEaseInOut, animations: {
+                        collectionView.contentOffset.x = collectionView.contentOffset.x - 20
+                    }, completion: nil)
+                })
+                UserDefaults.standard.setValue((UserDefaults.standard.value(forKey: "nudgeCount") as! Int) + 1, forKey: "nudgeCount")
+                UserDefaults.standard.setValue(false, forKey: "didFireNudge")
+            }
+        })
+    }
+    
+    
+    /* Dismiss the current pop up view */
+    func dismissPopUpView(marker: GMSMarker, fullyDismissed: Bool) {
+        UIView.animate(withDuration: 0.2, delay: 0.0, options: .curveEaseInOut, animations: {
+            self.popUpView.frame = CGRect(x: 12, y: self.view.bounds.height , width: self.view.bounds.width - 24, height: 130)
+        }, completion: { _ in
+            //remove from view after they animate off screen
+            self.popUpView.subviews.forEach({$0.removeFromSuperview()})
+            self.popUpView.removeFromSuperview()
+            if fullyDismissed {
+                self.currentMarker = nil
+            } else {
+                self.currentMarker = marker
+                self.popUp(stop: marker.userData as! Stop)
+            }
+        })
+    }
+    
+    
+    //MARK: collectionView Datasource
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as! CustomTimeCell
+        let currentDay = Days.fromNumber(num: getDayOfWeek(today: Date()))
+        if selectedStop.days.contains(currentDay!){
+            //MARK: After Kevin and I merge, go back and fix it so it only displays the shuttle times still to come. Example: If it is 12:45 -> only display shuttle buses from 12:45 and later for each marker
+            cell.textLabel.text = selectedStop.times[indexPath.row].shortDescription
+            cell.textLabel.center = CGPoint(x: cell.bounds.midX, y: cell.bounds.midY)
+        } else {
+            cell.textLabel.text = "No Shuttles Running Today"
+            cell.textLabel.center = CGPoint(x: cell.bounds.midX, y: cell.bounds.midY)
+            cell.textLabel.sizeToFit()
+        }
+        
+        return cell
+    }
+    
+    
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
```

Comment:
Don't think you need this function since I believe it returns 1 as default but try deleting it and see if it builds